### PR TITLE
feat: menu and footer [NCS-11]

### PR DESCRIPTION
**Pull request by anamontiaga**

### CODE REVIEW

File: src/components/modal/component.tsx [hunk 1]
```diff
@@ -47,40 +46,38 @@ export const Modal = (props: ModalProps) => {
     <FloatingPortal>
       <AnimatePresence>
         {open && (
-          <FloatingOverlay lockScroll>
```
Comment (mbarrenechea) [inline src/components/modal/component.tsx]: why did you remove the FloatingOverlay?

File: src/containers/home/menu/component.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import Wrapper from 'containers/wrapper';
+
+import Button from 'components/button';
+import { BREAKPOINTS } from 'styles/styles.config';
+
+import { NAV_OPTIONS } from './constants';
+
+const Menu: React.FC = () => {
+  const { minWidth } = useBreakpoint(BREAKPOINTS, 'md');
+  const [openModal, setOpenModal] = useState(false);
```
Comment (mbarrenechea) [inline src/containers/home/menu/component.tsx]: you have a hook for handling the state of modals. `useModal`

**Review by mbarrenechea — APPROVED** [review]

File: src/components/modal/component.tsx (hunk 1 above)
Comment (anamontiaga) [reply src/components/modal/component.tsx]: Has a position fixed that prevents the full screen modal from positioning itself correctly

File: src/containers/home/menu/component.tsx (hunk 2 above)
Comment (anamontiaga) [reply src/containers/home/menu/component.tsx]: I think this is outdated as ui store has been use to manage container modals
